### PR TITLE
PIC-2801

### PR DESCRIPTION
**Pull request by vb-moj**
. Ran bash script to rename project directories.

### CODE REVIEW

File: .circleci/config.yml-z
Comment (rschallapalli) [inline .circleci/config.yml-z:1]: Is this supposed to be removed?

File: hmpps-prepare-a-case-e2e.iml
Comment (rschallapalli) [inline hmpps-prepare-a-case-e2e.iml:1]: Needs a .gitignore

File: .circleci/config.yml-z
Comment (vb-moj) [reply .circleci/config.yml-z:1]: we are only using the dev namespace for e2e. Will discuss this latter today in the meeting.

File: server/utils/nunjucksSetup.ts
Comment (rschallapalli) [inline server/utils/nunjucksSetup.ts:1]: We wouldn't any nunjucks related artefacts as this app won't be having any front end

File: .circleci/config.yml-z
Comment (rschallapalli) [inline .circleci/config.yml-z:1]: we're supposed to remove this?

File: server/utils/nunjucksSetup.ts
Comment (vb-moj) [reply server/utils/nunjucksSetup.ts:1]: A lot of default tests failed when trying to remove this. I think it needs a proper FE dev should have a look and remove any un wanted files.

**Review by rschallapalli — APPROVED** [review]

**Pull request merged**